SIGS.md

```diff
@@ -0,0 +1,184 @@
+- Major cross-SIG decisions require project maintainer approval
+- All SIG work follows the project's [contribution guidelines](CONTRIBUTING.md)
+
+## Active Special Interest Groups
+
+| SIG | Focus Area | Meeting Schedule | Documentation |
+|-----|------------|------------------|---------------|
+| **[SIG Inference Scheduler](#sig-inference-scheduler)** | Intelligent request routing, load balancing, and traffic management | Weekly Tuesdays 12:00 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=12pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1aKTJru43krjHP2ORayEEp4JP-N7dJL8S)<br>• [Project Repository](https://github.com/llm-d/llm-d-inference-scheduler/) |
+| **[SIG Benchmarking](#sig-benchmarking)** | Performance testing, benchmarking frameworks, and optimization | Weekly Thursdays 1:00 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=1pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1Hd-rCRLDbucl-LD0RlQwOCLqERWF-obT?usp=drive_link)<br>• [Project Repository](https://github.com/llm-d/llm-d-benchmark) |
+| **[SIG PD-Disaggregation](#sig-pd-disaggregation)** | Prefill/decode separation, distributed serving, and workload disaggregation | Weekly Tuesdays 1:30 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=130pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1jk7wtojsWNbYQVf7BY8BEvIg8FMRZV0q?usp=drive_link) |
+| **[SIG KV-Disaggregation](#sig-kv-disaggregation)** | KV caching, prefix caching, and distributed storage systems | Weekly Tuesdays 12:00 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=12pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1mFbzwEWL2-LvD21owgxlKRcQD0eSmcz6?usp=drive_link)<br>• [Project Repository](https://github.com/llm-d/llm-d-kv-cache-manager) |
+| **[SIG Installation](#sig-installation)** | Kubernetes integration, deployment tooling, and platform operations | Weekly Thursdays 11:00 AM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=11am)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1H-0Y8fXepzrYpcaUOBfuphn1Cl-gU0xr?usp=drive_link) |
+| **[SIG Autoscaling](#sig-autoscaling)** | Traffic-aware autoscaling, resource management, and capacity planning | Weekly Wednesdays 2:00 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=2pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1iDlTgpFPOrSQn7dWR3uCQLtqhz86HTAi?usp=drive_link) |
+| **[SIG Observability](#sig-observability)** | Monitoring, logging, metrics, and operational visibility | Weekly Thursdays 12:30 PM ET<br>([Convert to your TZ](https://dateful.com/convert/eastern-time-et?t=12:30pm)) | • [Meeting Recordings and Docs](https://drive.google.com/drive/folders/1H-TVTCKYVxUn4fER7xuTPmscNttZCutN?usp=drive_link) |
+
+## SIG Detailed Descriptions
```

Comment:
It might be helpful to add GitHub handles for the leads so that the community knows who to reach out to from GitHub?